mikeio/generic.py

```diff
@@ -193,9 +203,11 @@ def diff(infilename_a, infilename_b, outfilename):
 
             itemdata_a = dfs_i_a.ReadItemTimeStep(item + 1, timestep)
             d_a = to_numpy(itemdata_a.Data)
+            d_a[d_a == deletevalue] = np.nan
```

Comment:
Use:
nan_indices = np.isclose(data, self._dfs.FileInfo.DeleteValueFloat, atol=1e-36)
data[nan_indices] = np.nan

??

Reply:
is this necessary? can we decide on a "atol" which will always work? What is fastest if the files are huge ?